Library/Formula/bork.rb

```diff
@@ -0,0 +1,18 @@
+class Bork < Formula
+  desc "Bash DSL for config management"
+  homepage "https://github.com/mattly/bork"
+  url "https://github.com/mattly/bork/archive/v0.9.1.tar.gz"
+  sha256 "01ff60779582d5ec67080d43a6325d3f5f213dd3dbbbd1ba0989e0253bdc73aa"
+
+  def install
+    prefix.install %w[bin docs lib test types]
+  end
+
+  test do
+    assert_match "checking: directory #{testpath}/foo\r" \
```

Comment:
Yeah, I see.  This works, but it needs a trailing space at the end of the `missing` line, which some editors will strip, so I think we'll just stick with what you've got:
```ruby
  test do
    expected = "checking: directory #{testpath}/foo\r"
    expected += <<-EOS.undent
      missing: directory #{testpath}/foo 
      verifying : directory #{testpath}/foo
      * success
    EOS
    assert_match expected, shell_output("#{bin}/bork do ok directory #{testpath}/foo", 1)